.github/workflows/cron_jerigon_zero_testing.yml

```diff
@@ -0,0 +1,102 @@
+name: Jerigon Zero Testing
+
+on:
+  # Uncomment when ready to run on a schedule
+  # schedule:
+  #   # Run every Sunday at 12:00 AM (UTC)
+  #   - cron: "0 0 * * 0"
+  push:
+    branches: [develop]
+  pull_request:
+    branches:
+      - "**"
```

Comment:
This will be removed when PR is debugged, right?

Reply:
yes, this is just for testing